Add lightweight comments to conf.py and update docs readme (#126100)
python/cpython: Doc/README.rst

Bugs in the toolset should be reported to the tools themselves.

> If you want to help the Documentation Team, you are always welcome.
> Leave a message on
> `discuss.python.org <https://discuss.python.org/c/documentation>`_.

Reviewer: I don't think there is a formal "Documentation Team" as such, this is perhaps more direct?:
```suggestion
To help with the documentation, or report any problems, please leave a message
on `discuss.python.org <https://discuss.python.org/c/documentation>`_.
```
